addons/hw_drivers/controllers/driver.py

```diff
@@ -102,9 +109,15 @@ def connect_box(self, token):
         image = get_resource_path('hw_drivers', 'static/img', 'False.jpg')
         if server == '':
             token = b64decode(token).decode('utf-8')
-            url, token = token.split('|')
+            try:
+                # IoT Box send token with db_uuid and enterprise_code only since V13
+                url, token, db_uuid, enterprise_code = token.split('|')
+                subprocess.check_call([get_resource_path('point_of_sale', 'tools/posbox/configuration/add_credential.sh'), db_uuid, enterprise_code])
```

Comment:
you should porbably write the scripts 'add_credential.sh' and 'load_certificate.sh' in python instead of shell script and like in page  'addons/hw_posbox_homepage/controllers/main.py'

it will avoid to have the logic everywhere.